.github/workflows/verify-generation-config.yaml

```diff
@@ -30,3 +72,40 @@ jobs:
           -e "REPO_BINDING_VOLUMES=${repo_volumes}" \
           gcr.io/cloud-devrel-public-resources/java-library-generation:latest \
```

Comment:
Ideally we should not use `latest`, but rather a versioned docker image, otherwise it is not hermetic. But since this is the first time we use the scripts and there might be bug fixes in this cycle, let's keep it for now. Once the scripts are stable, we should read the gapic-generator-java version from the build config file and use that version for the docker image.  
For now, can we extract `latest` as a parameter and share it between this step and the step below?

Reply:
Just FYI, we have a release job that releases `gcr.io/cloud-devrel-public-resources/java-library-generation:${GGJ_VERSION}`, so if we are to remove it, we can use that tag

Reply:
> Once the scripts are stable, we should read the gapic-generator-java version from the build config file and use that version for the docker image.

I think we should group the docker image tag with generator update so that renovate bot can update them together.

> For now, can we extract latest as a parameter and share it between this step and the step below?

Sounds good.